src/bg_window.c

```diff
@@ -39,7 +39,7 @@ static void ScrollWindow4bpp(Window *window, u8 direction, u8 y, u8 fillValue);
 static void ScrollWindow8bpp(Window *window, u8 direction, u8 y, u8 fillValue);
 static void BgConfig_HandleScheduledBufferTransfers(BgConfig *bgConfig);
 static void BgConfig_HandleScheduledScrolls(BgConfig *bgConfig);
-static void Bg_SetAffineScale(Background *bg, enum BgPosAdjustOp op, fx32 value);
+static void Bg_SetAffineScale(Background *bg, enum BgPosAdjustOp op, int value);
```

Comment:
no enum

Reply:
there are several enums in bg_window that aren't typedeffed. fixing this is out-of-scope.

Reply:
fair, note to self for me to fix this